### PR TITLE
Bug fix/visualizer exceed width

### PR DESCRIPTION
**Pull request by manuelaguirre**
adjusted graphics to get better visualization and made it scrollable when tree size exceeds the container's dimensions

### CODE REVIEW

File: src/components/shared/Visualizer.vue [hunk 1]
```diff
@@ -209,11 +233,9 @@ export default {
...
+    height: 600px;
     align-items: center;
   }
-  .svg {
-    width: 100%;
-    height: 100%;
-  }
 </style>
+
+//problema de render inicial y altura
```
Comment (sebastianfdez) [inline src/components/shared/Visualizer.vue:241]: Delete this :)

File: src/components/shared/Scrollbar.vue [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<template>
```
Comment (sebastianfdez) [inline src/components/shared/Scrollbar.vue:1]: Are you using this component ?

File: src/components/shared/Visualizer.vue [hunk 3]
```diff
@@ -130,9 +130,32 @@ export default {
       return undefined;
     },
     tree() {
-      return d3.tree().size([this.svgWidth, this.settings.width - 300]).separation(() => (this.settings.keyCellWidth * 2));
+      return d3.tree().nodeSize([this.settings.keyCellWidth, -this.settings.keyCellHeight * 2])
+        .separation((a, b) => (((a.data.leaves.keys.length + b.data.leaves.keys.length)) / 2 + (a.parent === b.parent ? 0.5 : 1)));
+    },
+    leftmostLeaf() {
+      const leaves = this.root ? this.root.leaves() : undefined;
```
Comment (sebastianfdez) [inline src/components/shared/Visualizer.vue:137]: maybe `firstLeaf()` or `leftLeaf()` but the most is a little redundant

File: src/components/shared/Scrollbar.vue (hunk 2 above)
Comment (manuelaguirre) [reply src/components/shared/Scrollbar.vue]: deleted it

Comment (manuelaguirre) [reply src/components/shared/Scrollbar.vue]: deleted it

**Pull request merged**